### DISCUSSION TITLE
wire:loading does not stay on File Upload

### DISCUSSION THREAD

**Discussion by AdamEsterle** [livewire/livewire#6738] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Upload a file on a form
2. put `wire:loading.attr="disabled"` on your button
3. When you select a file, it will disable, then quickly return to normal and then disable again and quickly return to normal.
4. Using S3 for file upload

This happens the first time when you select the file (is starts uploading to temp directory). Then it does it again after the file is fully uploaded to the temp dir. It used to stay disabled the entire upload process. But now it seems to disable itself, very quickly, only at the start and end of the upload process. During time it takes to upload behind the scenes, it is in a normal (not disabled) button state

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by AdamEsterle** [comment]
UPDATE:
It will work (like in Livewire 2.x) and stay disabled during the entire upload process if you include wire:target="video". It would be nice to not need a target and just have the button be disabled on any and all server requests (and stay disabled during file upload)

**Comment by Mushr0000m** [comment]
Stil having this issue and on a page with 10 files upload it's a pain to list them all in `target` it was working fine in v2
